src/firebolt/common/util.py

```diff
@@ -1,7 +1,8 @@
-from asyncio import get_event_loop, new_event_loop, set_event_loop
-from functools import lru_cache, wraps
+from functools import lru_cache, partial, wraps
 from typing import TYPE_CHECKING, Any, Callable, Type, TypeVar
 
+from trio import run as sync_run
```

Comment:
We still use asyncio for async stuff. Please rebase on the latest master

Reply:
Oops! I had merged into the wrong local branch.